### DISCUSSION TITLE
dependency injection this week

### DISCUSSION THREAD

**Discussion by itafra-net** [IT-Afra/bakery-shop#62] [Q&A] [answered]
Can we solve the issues with repositories and dependency injection this week to enhance the project?

**Comment by Mohsenarg** [comment, marked as answer]
I will do my best to solve the problem as soon as possible this week.

**Reply by itafra-net** [reply]
Thank you very much, Mohsen

**Comment by mohammadpirzadeh** [comment]
We will definitely solve this problem this week.

**Reply by itafra-net** [reply]
Thank you very much, Mohmmad
